src/core/resolvers/arco.ts

```diff
@@ -143,6 +143,12 @@ export interface ArcoResolverOptions {
    * @default false
    */
   resolveIcons?: boolean
+  /**
+   * disable style import
+   *
+   * @default false
+   */
+  disableSideEffect?: false
```

Comment:
```suggestion
  sideEffect?: boolean
```

and default to `true`

Reply:
好的